src/components/editor/metaform-editor-right-drawer-visibility.tsx

```diff
@@ -29,7 +30,7 @@ const MetaFormRightDrawerVisibility: FC<Props> = ({
 }) => {
   const [ selectedVisibleIf, setSelectedVisibleIf ] = React.useState<FieldRule | undefined>();
   const [ visibleIfSource, setVisibleIfSource ] = React.useState<VisibilitySource>(VisibilitySource.NONE);
-  const [ showConditions, setShowAndConditions ] = React.useState<boolean>(false);
+  const [ showConditions, setShowAndConditions ] = React.useState<number | null | undefined>();
```

Comment:
Check naming (should be consistent). Maybe should say like showVisibleIfProperties?